telemetry/src/ws.rs

```diff
@@ -29,12 +28,12 @@ const INTERNAL_CHANNEL_CAPACITY: usize = 10;
 /// # Errors
 /// Fails if unable to connect to the server
 pub async fn start(
-    RegularTelemetryConfig {
+    RegularTelemetry {
```

Comment:
```suggestion
    Telemetry {
```

why don't we just call it this?

Reply:
Renamed everywhere to just `telemetry`